### PR TITLE
Add large type to gel-foundations

### PR DESCRIPTION
**Pull request by sareh**
Part of #1508

**Overall change:**  Add additional GEL Type sizes for large contexts (Atlas, Elephant, Imperial, Royal, Foolscap) Sizes here: https://www.bbc.co.uk/gel/guidelines/typography#additional-type-sizes-for-larger-contexts 

UX context: When pairing with UX, it was decided that to start with all non-Latin font-sizes and line heights will initially be the same as those for Latin. Then they will stress-test these types with different scripts and where relevant ask for changes in a separate issue. 

**Code changes:**

- Add additional GEL Type sizes for large contexts (Atlas, Elephant, Imperial, Royal, Foolscap) to the scripts
- Add the methods `getAtlas` etc. which can be used by others within components. 

**Testing Note: this first PR does not update Storybook with the typography! http://localhost:8180/?path=/story/typography--canon this needs to be carried out with an update to `psammead-styles` in a separate PR after this `gel-foundations` change has been merged.** There will be a second PR for this issue #1508 to be complete. 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/utilities/gel-foundations/src/scripts/burmese.js [hunk 1]
```diff
@@ -1,4 +1,74 @@
...
+    },
+  },
+  imperial: {
+    groupA: {
+      fontSize: '50',
+      lineHeight: '54',
+    },
+    groupB: {
+      fontSize: '64',
+      lineHeight: '72',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/burmese.js]:
```suggestion
      lineHeight: '104',
```

File: packages/utilities/gel-foundations/src/scripts/burmese.js [hunk 2]
```diff
@@ -1,4 +1,74 @@
...
+    },
+    groupD: {
+      fontSize: '96',
+      lineHeight: '104',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/burmese.js]:
```suggestion
      lineHeight: '144',
```

File: packages/utilities/gel-foundations/src/scripts/burmese.js [hunk 3]
```diff
@@ -1,4 +1,74 @@
...
+    },
+  },
+  royal: {
+    groupA: {
+      fontSize: '40',
+      lineHeight: '44',
+    },
+    groupB: {
+      fontSize: '52',
+      lineHeight: '60',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/burmese.js]:
```suggestion
      lineHeight: '84',
```

File: packages/utilities/gel-foundations/src/scripts/burmese.js [hunk 4]
```diff
@@ -1,4 +1,74 @@
...
+    },
+    groupD: {
+      fontSize: '76',
+      lineHeight: '84',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/burmese.js]:
```suggestion
      lineHeight: '116',
```

File: packages/utilities/gel-foundations/src/scripts/burmese.js [hunk 5]
```diff
@@ -1,4 +1,74 @@
...
+    },
+  },
+  foolscap: {
+    groupA: {
+      fontSize: '32',
+      lineHeight: '36',
+    },
+    groupB: {
+      fontSize: '40',
+      lineHeight: '44',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/burmese.js]:
```suggestion
      lineHeight: '66',
```

File: packages/utilities/gel-foundations/src/scripts/burmese.js [hunk 6]
```diff
@@ -1,4 +1,74 @@
...
+    },
+    groupD: {
+      fontSize: '56',
+      lineHeight: '60',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/burmese.js]:
```suggestion
      lineHeight: '86',
```

File: packages/utilities/gel-foundations/src/scripts/latin-with-diacritics.js [hunk 7]
```diff
@@ -3,6 +3,76 @@
...
+    },
+  },
+  imperial: {
+    groupA: {
+      fontSize: '50',
+      lineHeight: '54',
+    },
+    groupB: {
+      fontSize: '64',
+      lineHeight: '72',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/latin-with-diacritics.js]:
```suggestion
      lineHeight: '78',
```

File: packages/utilities/gel-foundations/src/scripts/latin-with-diacritics.js [hunk 8]
```diff
@@ -3,6 +3,76 @@
...
+    },
+    groupD: {
+      fontSize: '96',
+      lineHeight: '104',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/latin-with-diacritics.js]:
```suggestion
      lineHeight: '116',
```

File: packages/utilities/gel-foundations/src/scripts/latin-with-diacritics.js [hunk 9]
```diff
@@ -3,6 +3,76 @@
...
+    },
+  },
+  royal: {
+    groupA: {
+      fontSize: '40',
+      lineHeight: '44',
+    },
+    groupB: {
+      fontSize: '52',
+      lineHeight: '60',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/latin-with-diacritics.js]:
```suggestion
      lineHeight: '64',
```

File: packages/utilities/gel-foundations/src/scripts/latin-with-diacritics.js [hunk 10]
```diff
@@ -3,6 +3,76 @@
...
+    },
+    groupD: {
+      fontSize: '76',
+      lineHeight: '84',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/latin-with-diacritics.js]:
```suggestion
      lineHeight: '92',
```

File: packages/utilities/gel-foundations/src/scripts/latin-with-diacritics.js [hunk 11]
```diff
@@ -3,6 +3,76 @@
...
+    },
+  },
+  foolscap: {
+    groupA: {
+      fontSize: '32',
+      lineHeight: '36',
+    },
+    groupB: {
+      fontSize: '40',
+      lineHeight: '44',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/latin-with-diacritics.js]:
```suggestion
      lineHeight: '48',
```

File: packages/utilities/gel-foundations/src/scripts/latin-with-diacritics.js [hunk 12]
```diff
@@ -3,6 +3,76 @@
...
+    },
+    groupD: {
+      fontSize: '56',
+      lineHeight: '60',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/latin-with-diacritics.js]:
```suggestion
      lineHeight: '68',
```

File: packages/utilities/gel-foundations/src/scripts/tamil.js [hunk 13]
```diff
@@ -1,4 +1,74 @@
...
+      lineHeight: '124',
+    },
+  },
+  imperial: {
+    groupA: {
+      fontSize: '50',
+      lineHeight: '54',
+    },
+    groupB: {
+      fontSize: '64',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/tamil.js]:
```suggestion
      fontSize: '52',
```

File: packages/utilities/gel-foundations/src/scripts/tamil.js [hunk 14]
```diff
@@ -1,4 +1,74 @@
...
+      lineHeight: '72',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/tamil.js]:
```suggestion
      lineHeight: '70',
```

File: packages/utilities/gel-foundations/src/scripts/tamil.js [hunk 15]
```diff
@@ -1,4 +1,74 @@
...
+    },
+    groupD: {
+      fontSize: '96',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/tamil.js]:
```suggestion
      fontSize: '76',
```

File: packages/utilities/gel-foundations/src/scripts/tamil.js [hunk 16]
```diff
@@ -1,4 +1,74 @@
...
+      lineHeight: '104',
+    },
+  },
+  royal: {
+    groupA: {
+      fontSize: '40',
+      lineHeight: '44',
+    },
+    groupB: {
+      fontSize: '52',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/tamil.js]:
```suggestion
      fontSize: '40',
```

File: packages/utilities/gel-foundations/src/scripts/tamil.js [hunk 17]
```diff
@@ -1,4 +1,74 @@
...
+      lineHeight: '60',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/tamil.js]:
```suggestion
      lineHeight: '58',
```

File: packages/utilities/gel-foundations/src/scripts/tamil.js [hunk 18]
```diff
@@ -1,4 +1,74 @@
...
+    },
+    groupD: {
+      fontSize: '76',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/tamil.js]:
```suggestion
      fontSize: '56',
```

File: packages/utilities/gel-foundations/src/scripts/tamil.js [hunk 19]
```diff
@@ -1,4 +1,74 @@
...
+      lineHeight: '84',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/tamil.js]:
```suggestion
      lineHeight: '80',
```

File: packages/utilities/gel-foundations/src/scripts/tamil.js [hunk 20]
```diff
@@ -1,4 +1,74 @@
...
+    },
+  },
+  foolscap: {
+    groupA: {
+      fontSize: '32',
+      lineHeight: '36',
+    },
+    groupB: {
+      fontSize: '40',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/tamil.js]:
```suggestion
      fontSize: '30',
```

File: packages/utilities/gel-foundations/src/scripts/tamil.js [hunk 21]
```diff
@@ -1,4 +1,74 @@
...
+      lineHeight: '44',
+    },
+    groupD: {
+      fontSize: '56',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/tamil.js]:
```suggestion
      fontSize: '46',
```

File: packages/utilities/gel-foundations/src/scripts/tamil.js [hunk 22]
```diff
@@ -1,4 +1,74 @@
...
+      lineHeight: '60',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/tamil.js]:
```suggestion
      lineHeight: '68',
```

File: packages/utilities/gel-foundations/src/scripts/thai.js [hunk 23]
```diff
@@ -1,4 +1,74 @@
...
+      lineHeight: '124',
+    },
+  },
+  imperial: {
+    groupA: {
+      fontSize: '50',
+      lineHeight: '54',
+    },
+    groupB: {
+      fontSize: '64',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/thai.js]:
```suggestion
      fontSize: '52',
```

File: packages/utilities/gel-foundations/src/scripts/thai.js [hunk 24]
```diff
@@ -1,4 +1,74 @@
...
+      lineHeight: '72',
+    },
+    groupD: {
+      fontSize: '96',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/thai.js]:
```suggestion
      fontSize: '76',
```

File: packages/utilities/gel-foundations/src/scripts/thai.js [hunk 25]
```diff
@@ -1,4 +1,74 @@
...
+      lineHeight: '104',
+    },
+  },
+  royal: {
+    groupA: {
+      fontSize: '40',
+      lineHeight: '44',
+    },
+    groupB: {
+      fontSize: '52',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/thai.js]:
```suggestion
      fontSize: '40',
```

File: packages/utilities/gel-foundations/src/scripts/thai.js [hunk 26]
```diff
@@ -1,4 +1,74 @@
...
+      lineHeight: '60',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/thai.js]:
```suggestion
      lineHeight: '58',
```

File: packages/utilities/gel-foundations/src/scripts/thai.js [hunk 27]
```diff
@@ -1,4 +1,74 @@
...
+    },
+    groupD: {
+      fontSize: '76',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/thai.js]:
```suggestion
      fontSize: '56',
```

File: packages/utilities/gel-foundations/src/scripts/thai.js [hunk 28]
```diff
@@ -1,4 +1,74 @@
...
+      lineHeight: '84',
+    },
+  },
+  foolscap: {
+    groupA: {
+      fontSize: '32',
+      lineHeight: '36',
+    },
+    groupB: {
+      fontSize: '40',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/thai.js]:
```suggestion
      fontSize: '36',
```

File: packages/utilities/gel-foundations/src/scripts/thai.js [hunk 29]
```diff
@@ -1,4 +1,74 @@
...
+      lineHeight: '44',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/thai.js]:
```suggestion
      lineHeight: '54',
```

File: packages/utilities/gel-foundations/src/scripts/thai.js [hunk 30]
```diff
@@ -1,4 +1,74 @@
...
+    },
+    groupD: {
+      fontSize: '56',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/thai.js]:
```suggestion
      fontSize: '50',
```

File: packages/utilities/gel-foundations/src/scripts/thai.js [hunk 31]
```diff
@@ -1,4 +1,74 @@
...
+      lineHeight: '60',
```
Comment (DenisHdz) [inline packages/utilities/gel-foundations/src/scripts/thai.js]:
```suggestion
      lineHeight: '76',
```

**Review by DenisHdz — COMMENTED** [review]
LGTM, but I have been comparing all the values with the ones UX added in the spreadsheet and I found several differences.

**Review by Bopchy — COMMENTED** [review]
Other than Dennis' comment, looks good - LGTM

**Comment by sareh** [comment]
@DenisBBC Thanks for the detailed review with all of your suggestions, Denis! 
However, having checked with UX, they'd like for us to use the Latin type sizes until they have been fully stress-tested for the other scripts and to then add the corrected type sizes at that point.

**Review by DenisHdz — APPROVED** [review]
Thank you for confirming this 👍

**Review by Bopchy — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]

**Comment by sareh** [comment]
This has been UX reviewed and approved.

**Comment by paruchurisilpa** [comment]
Moving this to Ready for merge without testing as this would be tested once  #1508 is done

**Pull request merged**